### ISSUE TITLE
lrp for regression

### ISSUE THREAD

**Issue by bmayaci** [kaifishr/PyTorchRelevancePropagation#2]
Dear @KaiFabi ,

Thanks for this implementation. I think it is super cool! Can your implementation be used for the regression problem where there is only one output neuron representing a numerical value. How should I change the implementation to do this, if possible?

Looking forward to hearing your comment on this!

Thanks

**Comment by kaifishr** [comment]
Hi @bmayaci

I have never tried relevance propagation for a regression task. At the moment, however, I don't see why it shouldn't work.

In `lrp.py`, line 104, you probably don't need the softmax function when working on a regression problem:

`relevance = torch.softmax(activations.pop(0), dim=-1)  # Unsupervised`

In this case `relevance = activations.pop(0)` should be enough.
